### PR TITLE
Dev/user signup/alex

### PR DESCRIPTION
**Pull request by alexcouper**
This moves the profile page into Angular land.

There's no tests at the angular end added as part of this :/

It's unclear to me how we want to structure the angular world stuff.
eg. the oauths stuff in controllers.js should be in another function, but where should that live?
and the app.js list of providers is gradually going to get unmanageable.

Do you think that registration and login should be moved inside the Angular app or be templates rendered through flask?

### CODE REVIEW

**Comment by alexcouper** [comment]
I'm going to start working on repo configuration now, so have created a story branch from this branch.

File: deploystream/apps/users/views.py [hunk 1]
```diff
@@ -28,6 +21,12 @@ def before_request():
         g.user = User.query.get(session['user_id'])
 
 
+@mod.route('/me/')
+@requires_login
+def home():
```
Comment (txels) [inline deploystream/apps/users/views.py:26]: This is now a bit weird as this URL was initially an "HTML" view, and now is a "JSON" view - this is where users are redirected after login.

Login redirection should now be `/` or `/#/profile`, I guess... ?

Is there any reason for this URL to exist at all, though?

Comment (alexcouper) [inline deploystream/apps/users/views.py:26]: I was thinking it would be `/#/profile`. This PR includes that change I believe.

**Comment by txels** [comment]
All good, see PR #91 for the case I meant

**Pull request merged**